### PR TITLE
SPDX license identifiers and seL4 Foundation setup

### PR DESCRIPTION
**Pull request by lsf37**
Also clarified license of zigurrat method file

### CODE REVIEW

File: apps/sel4bench/src/ipc.c [hunk 1]
```diff
@@ -107,8 +100,8 @@ process_ipc_results(void *r)
         };
 
         functions[i] = (char *) benchmark_params[i].name,
```
Comment (lsf37) [inline apps/sel4bench/src/ipc.c:102]: It looks to me like this should be a semicolon rather than a comma.

**Review by kent-mcleod — APPROVED** [review]

**Comment by lsf37** [comment]
@ssrg-bamboo `test`

**Comment by ssrg-bamboo** [comment]
Hello, I'm a bot! I'll bring this PR into Trustworthy Systems and run some tests

**Review by xurtis — APPROVED** [review]

**Comment by lsf37** [comment]
Merged in db3af10

**Pull request closed without merging**